addons/portal/static/src/scss/bootstrap_overridden.scss

```diff
@@ -1,6 +1,3 @@
-// The contrast ratio to reach against white, to determine if color changes from "light" to "dark". Acceptable values for WCAG 2.0 are 3, 4.5 and 7.
-// See https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast
-$min-contrast-ratio: 3 !default;
```

Comment:
I guess you should rather keep the portal one and remove the website one, no? Or maybe even move it to `/addons/web/static/src/scss/bootstrap_overridden_frontend.scss`. Basically any frontend override that do not depend on a website / portal variable could be there (so probably should).

Reply:
Hello @qsm-odoo, website module overwriting the file `web/bootstrap_overridden_frontend.scss` by `website/bootstrap_overridden.scss` so removing `$min-contrast-ratio` from the website leading to the error 'undefined variable'.  Also removing it from the portal is not creating an issue as it will be overwritten by the website.

Reply:
I don't understand what you said, why would it lead to "undefined variable"? In which case? In any case, if you remove it from the portal, then the min-constrast-ratio is not overridden anymore when website is not installed, which is not what we want. What is the issue with moving the overridde in `bootstrap_overridden_frontend.scss` ?

Reply:
Hello @qsm-odoo, 
The case I am considering is keeping it in `web/frontend` and removing it from the portal and website. 
But few variables[1][2][3] in `website/bootstrap_overridden.scss` directly or indirectly depend on `$min-contrast-ratio`. Therefore, if we move `$min-contrast-ratio`, we will also have to move those dependent variables. Is it okay to move those variables?

[1]https://github.com/odoo/odoo/blob/90190314d31c94abf5d0ba3f40fb5e11c106455e/addons/website/static/src/scss/bootstrap_overridden.scss#L54
[2]https://github.com/odoo/odoo/blob/90190314d31c94abf5d0ba3f40fb5e11c106455e/addons/website/static/src/scss/bootstrap_overridden.scss#L106
[3]https://github.com/odoo/odoo/blob/90190314d31c94abf5d0ba3f40fb5e11c106455e/addons/website/static/src/scss/bootstrap_overridden.scss#L77-L84

Reply:
Oh, ok, good catch, I did not think of that :+1: Let's do it this way:

- in web/primary_variables.scss, declare a new variable `$o-frontend-min-contrast-ratio` and set it to the desired value
- in web/bootstrap_overridden.scss, set `$min-contrast-ratio` to `$o-frontend-min-contrast-ratio`
- Do the same (keep the existing one but use `$o-frontend-min-contrast-ratio` as value) in both website and portal bootstrap_overridden.scss but also add a comment explaining why we need the value to be defined although it will be defined by web)